src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -180,17 +179,45 @@ public void jobLogic () {
         }
     }
 
+    /**
+     * Once the job is complete, notify subscribers and provide feedback on the job creation status.
+     */
     @Override
-    public void jobFinished () {
+    public void jobFinished() {
+        if (!status.error) status.completeSuccessfully("New version saved.");
+        // Send notification to those subscribed to feed version updates.
+        NotifyUsersForSubscriptionJob.createNotification(
+            "feed-updated",
+            feedSource.id,
+            getNotificationMessage(feedSource.name, status, feedVersion));
+
+    }
+
+    /**
+     * Create notification message based on the feed version validation result.
+     */
+    public static String getNotificationMessage(String feedSourceName, Status status, FeedVersion feedVersion) {
```

Comment:
This method shouldn't be static. All of the arguments being sent to it are instance variables of the class.

Reply:
Updated.